src/strings/decode_stream.c

```diff
@@ -555,7 +555,7 @@ MVMString * MVM_string_decodestream_get_available(MVMThreadContext *tc, MVMDecod
 }
 
 /* Checks if we have the number of bytes requested. */
-MVMint64 MVM_string_decodestream_have_bytes(MVMThreadContext *tc, const MVMDecodeStream *ds, MVMint32 bytes) {
+MVMint64 MVM_string_decodestream_have_bytes(MVMThreadContext *tc, const MVMDecodeStream *ds, MVMint64 bytes) {
     MVMDecodeStreamBytes *cur_bytes = ds->bytes_head;
     MVMint32 found = 0;
```

Comment:
needs to be 64 I think

Reply:
Also it looks like I can't add comments to lines that aren't close to a line you've changed, but in MVM_string_decodestream_bytes_available we also need `available` as 64bit, that's line 575